### PR TITLE
feat: administratable smart contract

### PR DESCRIPTION
**Pull request by antmat**
This PR introduces smart contract which allows "administrator" to perform ownership changing operstions, but still do not allow "administrator" to perform buisness logic related calls marked as onlyOwner. This reduces risks of accidentially making ownable smart contracts with data inaccessible due to migration.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -348,7 +348,7 @@ sonm-core (0.4.11) unstable; urgency=low
     all available token params, not only reward and net difficulty.
   * Fixed: proper env params applying in Connor (#1474).
   * Fixed: restrict image push with KYC level (#1458).
-    This PR introduces ACL which checks KYC level and allow only users of a
```
Comment (sokel) [inline debian/changelog]: left outer changes?

**Comment by quasisamurai** [comment]
Please, lint your test file.

File: debian/changelog (hunk 1 above)
Comment (antmat) [reply debian/changelog]: mass replace, thank, I'll fix it

File: blockchain/source/contracts/test/Dummy.sol [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.23;
+import "../Administratable.sol";
```
Comment (quasisamurai) [inline blockchain/source/contracts/test/Dummy.sol]: Missing blank line

File: blockchain/source/contracts/test/Dummy.sol [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+contract Dummy is Administratable {
```
Comment (quasisamurai) [inline blockchain/source/contracts/test/Dummy.sol]: Somehow, mark this contract as test-enviroment part

File: blockchain/source/contracts/Administratable.sol [hunk 4]
```diff
@@ -0,0 +1,62 @@
+pragma solidity ^0.4.23;
+
+contract Administratable {
+    address public owner;
+    address public administrator;
+
+
+    event OwnershipRenounced(address indexed previousOwner);
+    event OwnershipTransferred(
```
Comment (quasisamurai) [inline blockchain/source/contracts/Administratable.sol]: No multi-string events, please

File: blockchain/source/contracts/test/Dummy.sol (hunk 3 above)
Comment (antmat) [reply blockchain/source/contracts/test/Dummy.sol]: done

**Pull request merged**